plaidml/core/internal.h

```diff
@@ -49,7 +49,8 @@ using Variant = std::variant<  //
     std::monostate,            // PLAIDML_VALUE_NONE
     plaidml_dim_expr,          // PLAIDML_VALUE_DIM
     plaidml_expr,              // PLAIDML_VALUE_EXPR
-    double,                    // PLAIDML_VALUE_FLOAT
+    double,                    // PLAIDML_VALUE_DOUBLE
+    float,                     // PLAIDML_VALUE_FLOAT
```

Comment:
These are meant to be "arbitrary precision constant values", so I don't think we should make a version for each bitwidth.